TangoRosStreamer/nodelet_manager/src/main/java/eu/intermodalics/nodelet_manager/TangoInitializationHelper.java

```diff
@@ -122,70 +122,70 @@ public static final int loadTangoSharedLibrary() {
         if (!(new File(basePath).exists())) {
             basePath = "/data/data/com.projecttango.tango/libfiles/";
         }
-        Log.i("TangoInitializationHelper", "basePath: " + basePath);
+        Log.i("TangoInitializationHelp", "basePath: " + basePath);
```

Comment:
Android studio complained on this. The tag can have maximum 23 characters. Would you prefer a different tag? E.g. just TangoInitialization?